packages/frontend/track/src/events.ts

```diff
@@ -237,6 +249,15 @@ const PageEvents = {
     },
     appTabsHeader: {
       $: ['tabAction'],
+      chat: [
```

Comment:
reconsider the segment module of ai events, they are in a mess

Reply:
current ai track events are chaos, and it may not be possible to complete all the changes in this pr
i will keep the error reporting, move the other changes to the next pr, and continue modifying in next major version